frontend-react/src/pages/support/Faq.tsx

```diff
@@ -68,60 +78,45 @@ export const Faq = () => {
             </p>
             <hr />
             <h3>
-                I just activated my ReportStream account, why can't I log in?
+                I just activated my ReportStream account, why can't I log in?{" "}
             </h3>
+
             <p>
-                ReportStream, as part of the{" "}
+                ReportStream shares some resources with other projects operated
+                by the CDC (e.g.{" "}
                 <a
                     className="usa-link"
-                    href="https://www.cdc.gov/surveillance/projects/pandemic-ready-it-systems.html"
                     target="_blank"
-                    rel="noreferrer noopener"
+                    href="https://www.simplereport.gov/"
                 >
-                    Pandemic-Ready Interoperability Modernization Effort (PRIME)
+                    SimpleReport
                 </a>
-                , shares some resources with other projects operated by the CDC.
+                ).
             </p>
             <p>
                 Some aspects of the registration process may contain references
-                to{" "}
-                <a className="usa-link" href="https://simplereport.gov">
-                    SimpleReport
-                </a>
-                , a PRIME project that ReportStream closely collaborates with.
-                To access your user account, be sure to log in at{" "}
-                <Link to="/login" key="login" className="usa-link">
+                to SimpleReport. To access your ReportStream user account, be
+                sure to log in at{" "}
+                <a
+                    className="usa-link"
+                    href="http://reportstream.cdc.gov/login"
```

Comment:
This is throwing a SonarCloud security alert -- `http` needs to be `https`

Reply:
This has been fixed and pushed up!